src/seismometer/seismogram.py

```diff
@@ -409,4 +436,15 @@ def get_binary_targets(self):
             if self._is_binary_array(self.dataframe[[pdh.event_value(target_col)]])
         ]
 
+    def get_ordinal_categorical_targets(self):
+        return [
+            pdh.event_value(target_col)
+            for target_col in self.target_cols
+            if self._is_ordinal_categorical_terget(target_col)
+        ]
+
+    def _is_ordinal_categorical_terget(self, target_col):
```

Comment:
```suggestion
    def _is_ordinal_categorical_target(self, target_col):
```

tegget -> target if we are going to keep that language.

Reply:
Changed to `_is_ordinal_categorical_metric`.